views/login/code.pug

```diff
@@ -18,7 +18,7 @@ block content
             <summary><span>Help with personal access code</span></summary>
             <p>Your code is on the personally-addressed letter. It is case insensitive.</p>
 
-        input#redirect(name='redirect', type='hidden', value='/login/success')
+        input#redirect(name='redirect', type='hidden', value='/login/sin')
```

Comment:
works for now

Reply:
I'm not sure what you mean by that. That we'll need to change it in the future?

Reply:
yeah, but there's nowhere else to redirect to for now.